tips/TIP-0027/tip-0027.md

```diff
@@ -0,0 +1,318 @@
+
+The Committe of human validators will be managed and compensated with a DAO infratructure where we can setup multisig requirements for any vote to conclude.
+
+An API to support the integration of the policy registry into dApps can be called from the open-source repository.
+
+When submitting a `policyId` to the registry the following keys should be defined.
+
+- `issuerId` must be the original issuerId UTXO string
+- `policyId` must be the unique UTXO string of the policyNFT
+- `policyName` must be a unique human identifiable string
+- `websiteUrl` must be an official website or social media page url that illustrates publicly the `policyId`
+
+```json
+{
+  "registration": {
+    "issuerId": "s345d6f7g8h9ji07gy8h99",
+    "policyId": "a3s45dr6ft7gy87hu98ji09",
+    "policyName": "My iNFT Collection",
+    "websiteUrl": "https://mywebsite.com/policyid"
+  }
+}
+```
+
+### Creator Royalties
+
+A system to support interoperable royalty payments across dApps. Allowing universal secondary market reward systems to be integrated through-out the eco-system. Integration of such systems is at the choosing of the dApp developer but is encouraged to support creator royalties.
+
+royalty addresses can be defined inside the royalty addresses
+```js
+// the key inside the royalties object must be a valid iota1 address where royalties will be sent too
+// the value must be a numeric decimal representative of the percentage required ie. 0.05=5%
+
+{
+  ...
+  "royalties": {
+    "iota1...a": 0.5
+  }
+}
+```
+
+In the event there are further royalty, multiple royalty addresses could be used in the form of an object where the address and percentage are mapped in a key value format inside the `royalties` field.
+
+```json
+{
+  ...
+  "royalties": {
+    "iota1...a": 0.25,
+    "iota1...b": 0.25,
+    "iota1...c": 0.25,
+    "iota1...d": 0.25
+  }
+}
+```
+
+The total decimal sum of all `royaltyPercentage` must never exceed 1 and is recommended not to exceed 0.5
+
+
+if `royalties` exists, it is iterated over the keys and then all the royalties are paid out till there are no keys left to iterate over.
+
+
+
+### NFT Schema
+
+For ease of development and interoperability between applications within the eco-system an extendable schema standard is defined to support ease of integration of NFTs for developers.
+
+Each schema is defined by three main keys:
+
+- `standard` – the standard model
+- `schema` – the defined schema type
+- `version` – the version of the standard
+
+
+**Universal schema**
+Each NFT schema should consist of a collection of universal keys to define key properties of the NFT
+
+The standard defined is:
+
+- `IRC27`
+
+
+The schema is defined by one of the following:
```

Comment:
this just feels way to limiting and I would be really careful introducing your own system for this.

Reply:
The schema is expandable and versioning is implemented from the offset, introducing schema definitions is extremely advantageous for interoperability across dApps in the ecosystem